### ISSUE TITLE
Quick seek gesture restart the song

### ISSUE THREAD

**Issue by andiandi13** [vanilla-music/vanilla#833]
There is an issue with the "10sec seek" gesture.

When I swipe too quickly like "up up up up up", the song restart..

I personnaly use it with swipe up/down gesture but the bug is present for other gestures.

N.B. : I also want to add the seek bar often warp to the begging even if the restart doesn't occur.

N.B.2 : The seeking seems to be a little bit slow comparing to other players, with a quite long silence. I don't know if you can do something for that, but it's not the main issue at all.

**Comment by adrian-bl** [comment]
Just tested this on my phone and can not reproduce it.
Does it also happen if you disable the swipe up gesture? Maybe your phone registers a 'phantom event' which happens to push the '|<' button?

wrt. NB2: You mean there is a silence while seeking manually or with the up gesture?
I can't hear any pause if i seek manually but the gesture has a ~0.5sec silence (probably due to the way how we pass the seek message internally around)

**Comment by andiandi13** [comment]
Yes it also happens with any gesture. e.g. you can set seek while taping cover and it'll happen if you tap it multiple times and quite quickly, you should try. It was the same on my old Xperia too.

NB2 : no no with a gesture of course, that's it, it's about half a second. It would be better if it was instant or almost.
